examples/gno.land/r/nt/commondao/z_8_a_filetest.gno

```diff
@@ -48,9 +48,5 @@ func main() {
 	}
 }
 
-// Output:
-// Count = 4
-// A
-// B
-// C
-// D
+// Error:
+// unauthorized
```

Comment:
Also this king of errors should be removed to let the test succeed with the expected output:

```suggestion
// Output:
// Count = 4
// A
// B
// C
// D
```

Same for other cases.